cropper/src/main/java/com/theartofdev/edmodo/cropper/CropImage.java

```diff
@@ -343,7 +344,8 @@ public static Uri getCaptureImageOutputUri(@NonNull Context context) {
     Uri outputFileUri = null;
     File getImage = context.getExternalCacheDir();
     if (getImage != null) {
-      outputFileUri = Uri.fromFile(new File(getImage.getPath(), "pickImageResult.jpeg"));
+      File pickImageFile = new File(getImage.getPath(), "pickImageResult.jpeg");
```

Comment:
Update the path to reflect changes suggested by other comments. Maybe abstract the file factory in a static method on the custom file provider class. And some unit tests never hurt.

(You're contributing to a solution used by thousands, it's not just a quick workaround in your app anymore. Make it so it doesn't break again in a month.)